command.go

```diff
@@ -1016,9 +1137,9 @@ func (cmd *StringStringMapCmd) Scan(dst interface{}) error {
 }
 
 func (cmd *StringStringMapCmd) readReply(rd *proto.Reader) error {
-	_, err := rd.ReadArrayReply(func(rd *proto.Reader, n int64) (interface{}, error) {
-		cmd.val = make(map[string]string, n/2)
-		for i := int64(0); i < n; i += 2 {
+	_, err := rd.ReadMapReply(func(rd *proto.Reader, n int64) (interface{}, error) {
```

Comment:
Let's not introduce more helpers like `ReadArrayReply`, `ReadMapReply`, `ReadReply` - they don't help much and only make code look more complex. E.g. the more straightforward code is

```go
n, err := rd.ReadMapLen()
if err != nil {
    return err
}

for i := int64(0); i < n; i++ {
}
```

It is more readable and even slightly faster. What do you think?